src/localize/languages/da.json

```diff
@@ -0,0 +1,35 @@
+{
+  "common": {
+    "version": "Version",
+    "invalid_configuration": "Ugyldig konfiguration.",
+    "entity_missing": "Påkrævet vejr-enhed mangler.",
+    "invalid_forecast_days": "forecast_days skal være større end 0."
+  },
+  "weather": {
+    "clear-night": "Klart",
+    "cloudy": "Overskyet",
+    "fog": "Tåge",
+    "hail": "Hagl",
+    "lightning": "Lyn	",
```

Comment:
can you please remove the trailing spaces?

Reply:
Done